src/model.ts

```diff
@@ -68,6 +68,11 @@ export interface Model<TSchema extends BaseSchema, TDefaults extends Partial<TSc
     options?: DistinctOptions
   ) => Promise<Flatten<WithId<TSchema>[TKey]>[]>;
 
+  exists: (
+    filter: Filter<TSchema>,
+    options?: Omit<FindOptions<TSchema>, 'projection' | 'limit' | 'sort' | 'skip'>
```

Comment:
this custom type should be mentioned in the docs above https://github.com/plexinc/papr/pull/288/files#diff-5e4bb17f57f0581c13412c97c01c8bfec0886e3769cfb845f314ab996d9be238R422

Reply:
Good catch. Do you have any suggestions for what that type description should look like? Would it just be the same as this Typescript definition?

Reply:
Yeah, the typescript type def is fine